go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -289,11 +287,32 @@ func (c *Controller) syncServices(key string) error {
 			// get the endpoints associated to the vip
 			eps := getLbEndpoints(endpointSlices, svcPort, family)
 			// Reconcile OVN, update the load balancer with current endpoints
-			if c.needsOVNLBUpdate(eps, service) {
-				if err := loadbalancer.UpdateLoadBalancer(lbID, vip, eps); err != nil {
-					c.eventRecorder.Eventf(service, v1.EventTypeWarning, "FailedToUpdateOVNLoadBalancer",
-						"Error trying to update OVN LoadBalancer for Service %s/%s: %v", name, namespace, err)
-					return err
+			if c.needsOVNLBUpdate(eps.IPs.List(), service) {
```

Comment:
this is the part I need to change on top of yours, because this will have the OVN LB reject we don't need acls and will not need this logic